src/Internal.AspNetCore.Sdk/sdk/Sdk.props

```diff
@@ -3,20 +3,8 @@
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     <_ReferencesInternalAspNetCoreSdk>true</_ReferencesInternalAspNetCoreSdk>
-
-    <!-- Suppress the message about using a preview version of the .NET Core SDK -->
-    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
-
-    <!-- Make error messages clickable in VS Code's console -->
-    <GenerateFullPaths Condition="'$(VSCODE_PID)' != ''">true</GenerateFullPaths>
-
-    <!-- Default to setting warnings as errors -->
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Does this file pick up settings from files/KoreBuild/msbuild/KoreBuild.RepoTasks.Sdk/Sdk/Sdk.props or files/KoreBuild/KoreBuild.Common.props? If not, why are these no longer needed?

Reply:
No, it doesn't import from there. This can be removed because we stopped using Internal.AspNetCore.Sdk as an "SDK". This was originally added so, when used as an 'SDK', Internal.AspNetCore.Sdk could add implicit references to API check and other dependencies.